templates/harbor.yaml

```diff
@@ -18,6 +18,9 @@ spec:
     name: harbor
     version: 1.3.1
   values:
+    {{- if .Values.harbor.adminPassword }}
+    harborAdminPassword: {{ .Values.harbor.adminPassword | quote }}
+    {{- end }}
```

Comment:
What about this, to fix the harbor chart's sin of making it too easy to use its publicly-published default password?

```
harborAdminPassword: {{ default (randAlphaNum 32) .Values.harbor.adminPassword | quote }}
```

Reply:
Sure